common/persistence/persistenceInterface.go

```diff
@@ -22,6 +22,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+//go:generate mockgen -copyright_file ../../LICENSE -package mock -destination mock/store_mock.go go.temporal.io/server/common/persistence ShardStore,TaskStore,HistoryStore,ExecutionStore,MetadataStore,ClusterMetadataStore,VisibilityStore,Queue
```

Comment:
according to my memory, you do not need to specify the target interface?

```
ShardStore,TaskStore,HistoryStore,ExecutionStore,MetadataStore,ClusterMetadataStore,VisibilityStore,Queue
```

Reply:
see reply below.